inference-engine/src/mkldnn_plugin/nodes/mkldnn_conv_node.cpp

```diff
@@ -28,20 +28,19 @@ using namespace InferenceEngine;
 
 bool MKLDNNConvolutionNode::isSupportedOperation(const std::shared_ptr<const ngraph::Node>& op, std::string& errorMessage) noexcept {
     try {
-        if (isDynamicNgraphNode(op)) {
-            errorMessage = "Doesn't support op with dynamic shapes";
-            return false;
-        }
-
         if (!ngraph::is_type<ngraph::op::v1::Convolution>(op) && !ngraph::is_type<ngraph::op::v1::GroupConvolution>(op)) {
             errorMessage = "Only opset1 Convolution and GroupConvolution operations are supported";
             return false;
         }
-        size_t ndims = op->get_input_shape(0).size();
+        size_t ndims = op->get_input_partial_shape(0).rank().get_length();
         if ((ndims < 4) || (ndims > 5)) {
             errorMessage = "Doesn't support 'data' input with rank: " + std::to_string(ndims);
             return false;
         }
+        if (op->get_input_partial_shape(1).is_dynamic()) {
```

Comment:
comment for 147 - 150 lines
please add check on dynamic shapes here
`isWino == false` if node is dynamic
https://github.com/openvinotoolkit/openvino/blob/c02d96d17e3ffd004897f7969d72d2df486c8b6e/inference-engine/src/mkldnn_plugin/nodes/mkldnn_conv_node.cpp#L147-L150

Reply:
Could you please elaborate why we need this check. Wino convolution works well at least corresponding SL test passed.

Reply:
I missed that wino format has only `weight` input which is always constant for dynamic case